### ISSUE TITLE
Launch sponsors

### ISSUE THREAD

**Issue by ultimike** [mherchel/fldc-issues#24]
John,

Mike H. asked me to assign this to you - here's a list of FLDC2016 sponsors that we have commitments from that we can add to the site for launch. 

Gold:
WebEnabled

Silver:
Acquia
True North Custom
Hot Sauce Design and Development
Purple Rock Scissors
DrupalEasy

Bronze:
OSTraining

The following have committed to a sponsorship, but haven't decided on a level yet:
CloudNYNE (ahem)
OS Training
Big Couch

Logos and URLs are here: https://drive.google.com/open?id=0BxMBEI6GEcNzOE9HUFRFWTdZcmM - you may have to grab some logos from last year's site.

Let me know if you need anything else.

thanks,  
-mike

**Comment by JCL324** [comment]
Reassigning back to Mike A. Only 1 logo and 1 company bio so can't do much.

**Comment by JCL324** [comment]
I have enabled and updated those that were from last year on the site: DevPanel, Acquia, Hot Sauce, True North.  However, you indicate that OS Training is Bronze, but also not decided on level?

**Comment by ultimike** [comment]
John,

Thanks for your help on this one. OSTraining is Bronze, it is confirmed.

I just tried to add DrupalEasy as a sponsor and when I submitted the node, I got a Pathauto error: https://www.evernote.com/l/AAOG4A6pxa1DKpZaK-_MJiPNGfBjO7m-C30 - once this is fixed, I'll go ahead and add DrupalEasy and Purple Rock Scissors.

I went ahead and published Florida Technical College as an in-kind sponsor.

thanks,  
-mike
